### ISSUE TITLE
Can wrap-multilines work with no-extra-parens turned on?

### ISSUE THREAD

**Issue by gmathieu** [yannickcr/eslint-plugin-react#86]
When [no-extra-parens](http://eslint.org/docs/rules/no-extra-parens) is enabled the following returns a warning:

```
    render: function () {
        return (
            <div>
                <span>Hello</span>
            </div>
        )
    },
```

Is it possible for these two rules to co-exist?

**Comment by yannickcr** [comment]
As I responded for a [similar issue](https://github.com/yannickcr/eslint-plugin-react/issues/76#issuecomment-100026164):

> Sadly, as far as I know, a plugin don't have the power to silent an error reported by an ESLint rule, the only way we have to do something approaching is to replicate the ESLint rule, "patch" it to fit our needs and use it instead of the ESLint one. It is not very clean and it can be time consuming to maintain the rule in sync with the ESLint one.

So, currently we cannot have these two rules enabled :(

**Comment by gmathieu** [comment]
No worries. Thanks for the quick response and creating this repo!

**Comment by dsernst** [comment]
https://github.com/eslint/eslint/issues/3034 covers this issue as well. One tip [suggested](https://github.com/eslint/eslint/issues/3034#issuecomment-122068987) by @michaelficarra is to use the following form:

```jsx
    render: function () {
        return 0,
            <div>
                <span>Hello</span>
            </div>
    },
```

It's a little weird, but it silences the `no-extra-parens` error and maintains the balanced line spacing.

**Comment by jackwilsdon** [comment]
It looks like this was fixed in eslint/eslint#7926 (v3.14.0+), you can now do the following;

```JSON
{
  "rules": {
    "no-extra-parens": [ "warn", { "ignoreJSX": "none|all|multi-line|single-line" } ]
  }
}
```

In the case of this issue, it sounds like you want `multi-line`;
> allows extra parentheses around no/all/multi-line/single-line JSX components.
